lms/djangoapps/courseware/views/views.py

```diff
@@ -38,8 +38,8 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey, UsageKey
 from openedx_filters.learning.filters import CourseAboutRenderStarted
-from pytz import UTC
 from requests.exceptions import ConnectionError, Timeout  # pylint: disable=redefined-builtin
+from pytz import UTC
```

Comment:
its weird that this moved around and weird that they moved out of alphabetical order. django.http and django.shortcuts also switched